core/src/main/scala/kafka/server/KafkaApis.scala

```diff
@@ -1463,6 +1470,223 @@ class KafkaApis(val requestChannel: RequestChannel,
     sendResponseMaybeThrottle(request, createResponse)
   }
 
+  def handleListAcls(request: RequestChannel.Request): Unit = {
+    authorizeClusterAction(request)
+    val listAclsRequest = request.body[ListAclsRequest]
+    authorizer match {
+      case None => {
```

Comment:
Hmm, can you be a bit clearer which brace is unnecessary?  I get syntax highlighting errors if I delete any of them.